CHANGELOG.asciidoc

```diff
@@ -39,6 +39,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 - Add config option for windows/perfmon metricset to ignore non existent counters. {pull}6432[6432]
 - Refactor docker CPU calculations to be more consistent with `docker stats`. {pull}6608[6608]
 - Update logstash.node_stats metricset to write data under `logstash.node.stats.*`. {pull}6714[6714]
+- Changed `state_container` metric from `cpu.*.nanocores` to `cpu.*.cores`.
+- Fixed typo in values for `state_container` `status.phase`, from `terminate` to `terminated`.
```

Comment:
Here we could perhaps use `copy_to` to copy the field for now?

Reply:
I used code as there is a change on the scale, so needed to perform the math